tests/regression_tests/external_moab/main.cpp

```diff
@@ -0,0 +1,96 @@
+#include "openmc/capi.h"
```

Comment:
For style consistency, can you run this file through `clang-format` using the `.clang-format` settings?